tests/assembly/option-nonzero-eq.rs

```diff
@@ -0,0 +1,29 @@
+// revisions: WIN LIN
+// [WIN] only-windows
+// [LIN] only-linux
+// assembly-output: emit-asm
+// compile-flags: --crate-type=lib -O -C llvm-args=-x86-asm-syntax=intel
+// min-llvm-version: 14
```

Comment:
That's the minimum version anyway.

Reply:
Thanks; removed it.

(As usual I'm just cargo-culting these directives from some other test 😬)